index.html

```diff
@@ -1,14 +1,259 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="\styles\style.css" rel="stylesheet" />
+</head>
+
+<body>
+  <section id="header">
+    <header>
+      <a href="/index.html">
+        <h1>HOME PAGE</h1>
+      </a>
+      <h6>The page for the home</h6>
+    </header>
+    <br>
+    <br>
+    <section class="line-break">
+      <div class="double-line"></div>
+    </section><br>
+    <section class="line-break">
+      <div class="double-line"></div>
+
+    </section>
+    <br>
+    <section class="subtitle">
+      <h3>Take a look at this amazing <b>Home Page</b></h3>
+    </section>
+    <div id="images">
+      <div class='img-container'><img src='\images\haldimilk.jpg' alt="Healthy" width="250" height="150" />
+        <span class="caption">Home Remedies</span>
+      </div>
+      <div class='img-container'><img src='\images\salad.jpg' alt="Healthy" width="250" height="150" />
+        <span class="caption">Healthy Recipes</span>
+      </div>
+      <div class='img-container'><img src='\images\hairmist.jpg' alt="Healthy" width="250" height="150" />
+        <span class="caption">DIY</span>
+      </div>
+    </div>
+    <br>
+    <a href="/learn-more.html">
+      <button class="tell-more">Tell me more</button></a>
+    <br>
+    <br>
+  </section>
+  <section class="line-break">
+    <div class="double-line"></div>
+  </section>
+  <section>
+    <!------Special- Quotes------>
+    <blockquote>
+      <p>“FOOD ITSELF IS MEDICINE!”
+        –ANCIENT SIDDHAS</p>
+      <marquee>
+        <h2>When diet is correct,Medicine is of no need</h2>
+      </marquee>
+      <!-----Audio Frame link from soundcloud-->
+      <iframe width="10%" height="50" scrolling="no" frameborder="no" allow="autoplay"
+        src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/968008882&color=%23ff5500&auto_play=true&hide_related=false&show_comments=true&show_user=true&show_reposts=false&show_teaser=true"></iframe>

```

Comment:
styling this a little differently could help keep the layout of the page looking nicer. We could adjust the player to be a sticky header in the top right or left corner that would scroll with the page and be styled to look more coherent to make it fit better. But nice work imbedding music to your page using an iframe!